### PR TITLE
[@mantine/tiptap]: add forward ref to mantine tip tap editor

### PR DESCRIPTION
**Pull request by adarshaacharya**
- forward ref to avoid ref warning in mantine tip tap editor

### CODE REVIEW

File: src/mantine-tiptap/src/RichTextEditor.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ const defaultProps: Partial<RichTextEditorProps> = {
   withTypographyStyles: true,
 };
 
-export function RichTextEditor(props: RichTextEditorProps) {
+export const RichTextEditor = forwardRef<HTMLInputElement, RichTextEditorProps>((props, ref) => {
```
Comment (rtivital) [inline src/mantine-tiptap/src/RichTextEditor.tsx:53]: ref type is incorrect, it is div element, not input

Comment (adarshaacharya) [reply src/mantine-tiptap/src/RichTextEditor.tsx]: corrected, thanks

**Comment by rtivital** [comment]
Thanks!

File: src/mantine-tiptap/src/RichTextEditor.tsx (hunk 1 above)
Comment (AabhasDhaubanja) [reply src/mantine-tiptap/src/RichTextEditor.tsx]: thanks adarsha

**Pull request merged**